### PR TITLE
[usecase-embeddingapi] Add tests for setJavaScript related APIs

### PR DESCRIPTION
**Pull request by haoxli**
Impacted tests(approved): new 4, update 0, delete 0
Unit test platform: Crosswalk for Android 21.51.546.0
Unit test result summary: pass 4, fail 0, block 0

BUG=https://crosswalk-project.org/jira/browse/CTS-1818

### CODE REVIEW

**Comment by haoxli** [comment]
@Shao-Feng PTAL

**Comment by Shao-Feng** [comment]
The README.md should be updated for new test case with the function it covered

**Comment by haoxli** [comment]
ok, I will make a separate PR for this PR and previous.

File: usecase/usecase-embedding-android-tests/embeddingapi/src/org/xwalk/embedded/api/sample/setting/XWalkViewSettingJavaScriptEnabled.java [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+            mButton.setText(ENABLE_BT);
+        }
+        mMessage.setText(MESSAGE_TITLE + defaultValue + "(Default)");
+        mXWalkView.load("file:///android_asset/javascript_enabled.html", null);
+
+        mButton.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                if (ENABLE_BT.equals(mButton.getText())) {
+                    settings.setJavaScriptEnabled(true);
```
Comment (haoxli) [inline usecase/usecase-embedding-android-tests/embeddingapi/src/org/xwalk/embedded/api/sample/setting/XWalkViewSettingJavaScriptEnabled.java]: should be setJavaScriptCanOpenWindowsAutomatically

File: usecase/usecase-embedding-android-tests/embeddingapi-async/src/org/xwalk/embedded/api/asyncsample/setting/XWalkViewSettingJavaScriptCanOpenWindowsAutomaticallyAsync.java [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+            mButton.setText(ENABLE_BT);
+        }
+        mMessage.setText(MESSAGE_TITLE + defaultValue + "(Default)");
+        mXWalkView.load("file:///android_asset/window_create_open.html", null);
+
+        mButton.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                if (ENABLE_BT.equals(mButton.getText())) {
+                    settings.setJavaScriptEnabled(true);
```
Comment (haoxli) [inline usecase/usecase-embedding-android-tests/embeddingapi-async/src/org/xwalk/embedded/api/asyncsample/setting/XWalkViewSettingJavaScriptCanOpenWindowsAutomaticallyAsync.java]: should be setJavaScriptCanOpenWindowsAutomatically

**Pull request merged**